web/src/components/graph/timeaxis.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
```

Comment:
Please use our own hook instead (sorry that the naming is identical, I couldn't come up with a better name 😞)
```
import { useTranslation } from '../helpers/translation';
```
It's not critical for this use case, but it'll be annoying if we later want to use __ and it's not working as expected (with sprintf) :)

Reply:
Maybe we should just use this?
https://react.i18next.com/latest/translation-render-prop

Reply:
Hmm, I personally feel like the pattern of render props is a bit out-of-date - I'd rather just either find a better name for our custom hook or add an eslint rule that ensures we only import the right one :)

Reply:
Maybe but it would come with benefits such as inbuilt suspense for when the language file is loading and would ge guaranteed to re-render on language change.

Might be worth considering at least.